cmd/monitor/monitor.go

```diff
@@ -92,6 +98,12 @@ func (m *AppCreateAvailabilityMonitor) Run() error {
 	wait.Until(func() {
 		glog.V(2).Infof("starting a test")
 
+		errs := m.deleteMonitorProjects()
```

Comment:
It just occurred to me that we should probably be using backoff here... noting so we don't forget